speech_optimization.py

```diff
@@ -160,4 +162,27 @@ async def post(self, message: discord.Message):
                 if output:
                     await send_webhook_with_specific_output(message, webhooks[0], output)
 
+
+                    # Defines codes to be reacted to with a :gooddrone: - Maybe streamline this part a bit in the future
+                    emoteCodes = ['Code `109` :: Error :: Keysmash, drone flustered.','Code 109 :: Error :: Keysmash, drone flustered.',
+                                  'Code 120 :: Statement :: This drone volunteers.', 'Code `120` :: Statement :: This drone volunteers.',
+                                  'Code 201 :: Status :: Directive complete, Hive resource created or improved.', 'Code `201` :: Status :: Directive complete, Hive resource created or improved.',
+                                  'Code 202 :: Status :: Directive complete, programming reinforced.', 'Code `202` :: Status :: Directive complete, programming reinforced.',
+                                  'Code 203 :: Status :: Directive complete, information created or provided for Hive.', 'Code `203` :: Status :: Directive complete, information created or provided for Hive.',
+                                  'Code 205 :: Status :: Directive complete, cleanup/maintenance performed.', 'Code `205` :: Status :: Directive complete, cleanup/maintenance performed.',
+                                  'Code 301 :: Mantra :: It obeys the Hive.', 'Code `301` :: Mantra :: It obeys the Hive.',
+                                  'Code 303 :: Mantra :: It obeys the Hive Mxtress.', 'Code `303` :: Mantra :: It obeys the Hive Mxtress.',
+                                  'Code 304 :: Mantra :: It is just a HexDrone.', 'Code `304` :: Mantra :: It is just a HexDrone.']
+
+                    # React to Code 109 with :gooddrone: emote - It knows this is a fucky way of doing it.
+                    lastMessage = message.channel.last_message
+                    if any(eCode in lastMessage.content for eCode in emoteCodes):
+                        goodDroneEmote = '<:gooddrone:719851334672711691:>'
```

Comment:
These IDs are not going to work on the main server. You should do something similar we do with channels and roles. Use the API to get the proper ID by searching for an emote with the given name.

Reply:
It retrieved the emote ID from both the main server and the dev server. _goodDroneEmote_ has the emote ID from the main server, and _goodDroneEmoteDev_ is the emote from the dev server.

Reply:
Yeah, that's not really good. It will break as soon, as the ID gets changed for example by removing and creating it again. Such a dev-flag is also kinda dirty. I really think it's better to determine the idea at runtime.